<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoeffWindow</class>
 <widget class="QMainWindow" name="CoeffWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление ставками</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icons.qrc">
    <normaloff>:/coeff.png</normaloff>:/coeff.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Стаж</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ставка</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="stageEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="coeffEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="save_coeff">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_coeff">
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_coeff">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="coeff_table"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>372</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
